README.md

```diff
@@ -59,5 +60,13 @@ helm delete --purge che-devfile-registry
 docker run -it --rm -p 8080:8080 quay.io/eclipse/che-devfile-registry:nightly
 ```
 
+## CI
+The following [CentOS CI jobs](https://ci.centos.org/) are associated with the repository:
+
+- [`master`](https://ci.centos.org/job/devtools-che-devfile-registry-build-master/) - builds CentOS images on each commit to the [`master`](https://github.com/eclipse/che-devfile-registry/tree/master) branch and pushes them to [quay.io](https://quay.io/organization/eclipse).
+- [`nightly`](https://ci.centos.org/job/devtools-che-devfile-registry-nightly/) - builds CentOS images and pushes them to [quay.io](https://quay.io/organization/eclipse) on a daily basis from the [`master`](https://github.com/eclipse/che-devfile-registry/tree/master) branch.
```

Comment:
I would add that's the version used by default by the nightly version of che. That's the main reason we do a nightly build even if we do continuous builds.

Reply:
@l0rd good point. what is the job that does the Che nighlty builds? Can not find anything nightly specific here - https://ci.codenvycorp.com/

Reply:
I think it's https://ci.codenvycorp.com/view/nightly-builds/job/all-che-docker-images-nightly/